main/webapp/modules/core/langs/translation-en.json

```diff
@@ -285,6 +285,8 @@
     "core-dialogs/for-null-cell-value-to-empty-str-label": "Convert null value to NULL in INSERT",
     "core-dialogs/choose-export-destination": "Please choose the destination for project export",
     "core-dialogs/export-to-local": "OpenRefine project archive to file",
+    "core-facets/facet-current-title": "The current name of the facet is: $1.\n To what name do you want to change it?",
```

Comment:
I would make the question even simpler: instead of "To what name do you want to change it?", I would just use "Enter the new name" or something like that.